app/layout.tsx

```diff
@@ -34,15 +36,15 @@ export default function RootLayout({
   const contentCss = 'content bg-slate-100 px-10 lg:px-0 grow overflow-y-auto';
 
   return (
-    <html>
+    <html suppressHydrationWarning>
```

Comment:
here is to avoid warning locally. it seems that the language got injected while loading, which produces an inconsistency with the html produced by the server. We could remove it but it does not do harm. it affects ONLY this tag (no children).